lib/simulator-xcode-9.js

```diff
@@ -51,8 +52,14 @@ class SimulatorXcode9 extends SimulatorXcode8 {
   }
 
   async shutdown () {
+    await restoreTouchEnrollShortcuts();
```

Comment:
OK, now I see that restoreTouchEnrollShortcuts has nothing to do with the UI. However it might still be an issue if we are running parallel tests